src/status_im2/contexts/quo_preview/graph/wallet_graph.cljs

```diff
@@ -0,0 +1,91 @@
+(ns status-im2.contexts.quo-preview.graph.wallet-graph
+  (:require [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(defn generate-crypto-token-prices
+  [num-elements volatility]
+  (loop [n          num-elements
+         prices     []
+         prev-price (rand-int 100000)
+         volatility volatility]
+    (if (zero? n)
+      (vec (reverse prices))
+      (let [fluctuation  (* prev-price volatility)
+            random-delta (- (rand fluctuation) (/ fluctuation 2))
+            new-price    (+ prev-price random-delta)
```

Comment:
You can probably replace these two lines to define `new-price` with `(max 1 (+ prev-price random-delta))`

Reply:
Done